intelmq/bots/BOTS

```diff
@@ -307,6 +307,41 @@
                 "rate_limit": 10800
             }
         },
+        "Generic Mail Attachment Fetcher": {
+            "description": "Monitor IMAP mailboxes and retrieve mail attachments",
+            "module": "intelmq.bots.collectors.mail.collector_mail_attach",
+            "parameters": {
+                "accuracy": "",
```

Comment:
While this is a string, it's default is defined in `defaults.conf` and can be overwritten on per-bot basis. You are overriding this default with `None`. Just remove this line for both bots.